Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fuzzers/afl:2.52
```

Comment:
Please don't use git clone here. In fact I don't think you need to do wget either (since the repo is already in the Docker image). I'm not sure I understand the purpose of LD_LIBRARY_PATH being set. Finally, please include a package stage.

Reply:
these are a little bit old harnesses, need indeed refactoring. LD_LIBRARY_PATH is required because when run in Mayhem it fails because can not find the lib3mf shared object. Mayhem also recommends to set it.